test/cpp/api/module.cpp

```diff
@@ -343,15 +343,22 @@ TEST_F(ModuleTest, Conversion_MultiCUDA) {
     }
   }
   {
-    module->to(torch::kInt32);
+    module->to(torch::kFloat64);
     for (auto& parameter : module->parameters()) {
-      ASSERT_EQ(parameter.dtype(), torch::kInt32);
+      ASSERT_EQ(parameter.dtype(), torch::kFloat64);
     }
   }
+}
+
+TEST_F(ModuleTest, Conversion_MultiCUDA_NoGrad) {
```

Comment:
My guess as to why this test is failing is that you should rename this to `Conversion_NoGrad_MultiCUDA` (so that it ends with `CUDA`) or else this test will also be run for non-CUDA builds.